### ISSUE TITLE
multiple linters How to set individually 

### ISSUE THREAD

**Issue by DMC-HYX** [SublimeLinter/SublimeLinter#1856]
i have many linters e.g php,phpmd,phpcs,phpins,golint。but I do not want each that “show_panel_on_save:view” ;

i want setting like the following. all default "show_panel_on_save:view“ ，linters.phpmd setting "show_panel_on_save:never“

![图片](https://user-images.githubusercontent.com/22950085/182573926-dd0cf62a-368a-4a27-88de-f46db50b7a05.png)

There are also ”.pdepend“ folder that keep adding up until the disk is full, and I have to delete it

**Comment by kaste** [comment]
Yeah sure, we don't have that and if you're interested in it, you should try an implementation.  

PS. I don't think ".pdepend" is coming from here.  That's just another tool you're using.
